<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.qYGCuoQ' shared service.

return $this->privates['.service_locator.qYGCuoQ'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('fileUploader' => function (): \App\Service\QuestionImageUploader {
    return ($this->privates['App\Service\QuestionImageUploader'] ?? $this->privates['App\Service\QuestionImageUploader'] = new \App\Service\QuestionImageUploader(($this->targetDirs[3].'/public/uploads/question_images')));
}, 'quiz' => function (): \App\Entity\TrainingQuiz {
    return ($this->privates['.errored..service_locator.qYGCuoQ.App\Entity\TrainingQuiz'] ?? $this->load('getTrainingQuizService.php'));
}));
